submissions/adekemiAwomolo/PhpAndDtabase/customersView.php

```diff
@@ -0,0 +1,61 @@
+<?php
+require_once('connection.php');
+
+    try{
+    global $pdo;
+    $sql = "SELECT  * FROM orders";
```

Comment:
To get all the details expected for you for this task. You need to use one of the `JOIN` statements to connect to all the tables that relationships exist. Just like how you have been taught doing your database task

Reply:
The query is wrong